kolibri/content/management/commands/populate_channel.py

```diff
@@ -0,0 +1,22 @@
+'''
+to run the command: <kolibri manage populate_channel>
+'''
+from django.core.management.base import BaseCommand
+from kolibri.content import models
+
+
+class Command(BaseCommand):
+    help = """To collect data for fixture, run the commandline <kolibri manage dumpdata content --exclude=contenttypes>.'
```

Comment:
I think our preference for docstrings is to use the convention outlined here:

http://thomas-cokelaer.info/tutorials/sphinx/docstring_python.html

@MCGallaspy @benjaoming correct me if I am wrong.